app/views/admin/users/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<ul>
+<% @users.each do |user| %>
+  <div id="users-<%= user.id %>-info">
+    <li>Name: <%= link_to user.name, admin_user_path(user) %></li>
+    <li>Member since: <%= Date.strptime(user.created_at.to_s) %></li>
+    <li><%= button_to "Upgrade to Merchant" %></li>
```

Comment:
#todo add the path to this button.

Reply:
It wasn't part of the User Story I did but I figured it would be in a later one.